utils/docker/Dockerfile.fodera

```
@@ -0,0 +1,324 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# All rights reserved.
+#
+# 'recipe' for Docker to build an image of centOS 8 based
+# environment for building the DAOS project.
+#
+
+# Pull base image
+ARG BASE_DISTRO=fedora:34
+FROM $BASE_DISTRO
```

Unless this is going to be modified on the command line you don't need to do this.
```suggestion
FROM fedora:34
```